### DISCUSSION TITLE
Help about generating certificate

### DISCUSSION THREAD

**Discussion by milletown** [Keyfactor/ejbca-ce#496] [General]
Hello

I am new in this forum. I am working with Ejbca to generate certificates.
Is there a solution to define a multi ip_address in the Altname  option please?
thanks for your help
thierry

**Comment by primetomas** [comment]
what is a "multi ip_address"?
You can add multiple IP adresses, both IPv4 and IPv6 in End Entity profiles, which you then can fill in this information during request in the RA UI (or Admin UI).

https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/end-entities-overview/end-entity-profiles-overview/end-entity-profiles-fields#EndEntityProfilesFields-IPaddress

Cheers,
Tomas
